src/somef/extract_software_type.py

```diff
@@ -0,0 +1,309 @@
+import os
+from pathlib import Path
+import nbformat
+from chardet import detect
+import re
+from .extract_workflows import is_file_workflow
+from .process_results import Result
+
+media_extensions = ('.rData','.bib','.csv','.xlx','.stan','.tex','.html','.css','.rst','.md','.png', '.jpg', '.jpeg', '.gif', '.bmp', '.svg', '.mp4', '.avi', '.mkv', '.wmv', '.flv', '.mov', '.mp3', '.wav', '.wma', '.flac', '.aac', '.ogg', '.pdf' )
+
+def check_repository_type(path_repo,title,metadata_result:Result):
+    """ Function that adds the metadata result in the JSON 
+        output depending on the software type"""
+    if check_extras(path_repo):
+        metadata_result.add_result(constants.TYPE,
+                                    {
+                                        constants.PROP_VALUE: 'https://www.britannica.com/technology/software',
+                                        constants.PROP_TYPE: constants.URL,
+
+                                    },
+                                    1,
+                                    constants.TECHNIQUE_HEURISTICS)
+    elif check_static_websites(path_repo):
+        metadata_result.add_result(constants.TYPE,
+                                    {
+                                        constants.PROP_VALUE: 'https://www.britannica.com/technology/software',
```

Comment:
Why not "Static Website"?

Reply:
As commented above we talked about putting the link instead of just the category, but i could modify to put the category instead

Reply:
Yes, it's more clear.